log10/load.py

```diff
@@ -214,9 +214,14 @@ async def log_async(completion_url, func, **kwargs):
         if "messages" in kwargs:
             kwargs["messages"] = flatten_messages(kwargs["messages"])
 
+        if "anthropic" in func.__module__:
+            if "system" in kwargs:
+                kwargs["messages"].insert(0, {"role": "system", "content": kwargs["system"]})
+
         log_row = {
             # do we want to also store args?
             "status": "started",
+            "kind": "chat" if "chat" in func.__module__ or "messages" in func.__module__ else "completion",
```

Comment:
seems completions needs `kind` to render in GUI. So set it when post the request. Otherwise, the completions only shows after response (with kind set) posted.